features/step_definitions/journey_test_steps.rb

```diff
@@ -1068,8 +1047,14 @@ def services_listed(service_ids)
   if page_name == 'Digital Marketplace'
     page.visit("#{dm_frontend_domain}")
     page.should have_content("#{page_name}")
+    page.should have_link('Find an individual specialist')
```

Comment:
Yes, expectation is that DOS will be live on preview to be able to start running the tests